easybuild/easyconfigs/s/Schrodinger/Schrodinger-2023-2.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'PackedBinary'
+
+name = 'Schrodinger'
+version = '2023-2'
+
+homepage = 'https://www.schrodinger.com/'
+description = """ Schrodinger aims to provide integrated software solutions and services 
+ that truly meet its customers needs. We want to empower researchers around the world to 
+ achieve their goals of improving human health and quality of life through advanced computational 
+ techniques that transform the way chemists design compounds and materials. """
+
+toolchain = SYSTEM
+
+download_instructions = 'Manual download from schrodinger.com'
+sources = ['%(name)s_Suites_%(version)s_Linux-x86_64.tar']
+checksums = ['40f24903fa2ca8fa2928ca6e72c71a3c0733302b4bb16b039df9aaf93d5469fc']
+
+install_cmd = 'cd %(builddir)s/%(name)s_Suites_%(version)s_Linux-x86_64 && '
+# Ignoring error code as schrodingers installer checks OS packages
+# https://raw.githubusercontent.com/schrodinger/linux_package_requirement/main/check_required_packages.sh
+# but their bundled liblzma they the the installer puts in LD_LIBRARY_PATH breaks librpmio.so
```

Comment:
Please fix this comment

Reply:
@Micket ping

Reply:
Don't know what is supposed to be fixed.

Reply:
Read that last line :-)